src/styles/paths.less

```diff
@@ -0,0 +1,17 @@
+.sn-org-chart {
+  .sn-org-svg {
+    position: absolute;
+
+    .sn-org-link {
```

Comment:
I think we should call them either links or paths, not both

Reply:
yea, paths it is